internal/generator/handlers.go

```diff
@@ -108,33 +111,32 @@ func (gen *generator) RPC(rpc *proto.RPC) {
 		}
 	}
 
-	_, reqExamples, resExamples, err := parseComment(rpc.Comment)
+	// NOTE: Redocly does not read the "examples" (plural) field, only the "example" (singular) one.
```

Comment:
So we can only have a single example? Is it adhering to the spec? https://swagger.io/specification/

Reply:
The spec allows the `example` to be a key-value object with multiple examples. 

```yaml 
example:
  "example one":
    Foo: 42"
  Two:
    Bar: Atreides
```